src/partition/repart.c

```diff
@@ -4907,6 +4931,12 @@ static int do_copy_files(Context *context, Partition *p, const char *root) {
                         (void) copy_xattr(sfd, NULL, tfd, NULL, COPY_ALL_XATTRS);
                         (void) copy_access(sfd, tfd);
                         (void) copy_times(sfd, tfd, 0);
+
+                        timespec_store_nsec(tspec + 0, UTIME_OMIT);
```

Comment:
Please do not use `timespec_store_nsec()` for storing special values like `UTIME_OMIT`.
Please do something similar in mkdir.c.